dropwizard-archetypes/src/test/resources/settings.xml

```diff
@@ -17,6 +17,17 @@
                         <enabled>true</enabled>
                     </snapshots>
                 </repository>
+                <repository>
+                    <id>ossrh</id>
+                    <name>Sonatype Nexus Snapshots</name>
+                    <url>@localRepositoryUrl@</url>
```

Comment:
This took a long time to figure out. Basically because we have the repository entry in the pom to fetch snapshots from ossrh, this also caused archetype builds to not use the dropwizard SNAPSHOT builds from the multi-module build and instead download the last published snapshot from Nexus. This meant that any changes made as part of this PR weren't visible to the archetype build (specifically, the jakarta.validation-api dependency added to dropwizard-dependencies BOM isn't in the published snapshot so the archetype build failed).

Reply:
It should work the same during a release build. This repository only allows snapshots and not releases.